website/static/playground.js

```diff
@@ -96,7 +96,7 @@ window.onload = function() {
             prettierVersion;
         } else {
           link.href =
-            "https://github.com/j-f1/forked-prettier/blob/master/CHANGELOG.md#" +
+            "https://github.com/prettier/prettier/blob/master/CHANGELOG.md#" +
             prettierVersion;
```

Comment:
We should omit `.` in this case.

Not working: [`https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1.9.2`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1.9.2)
Working: [`https://github.com/prettier/prettier/blob/master/CHANGELOG.md#192`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#192)

Reply:
D’oh! Fixed in https://github.com/prettier/prettier/pull/3564/commits/975aac1e8c9495cc5c8482bff5c368e81f0d3889.